sdk/src/androidTest/java/co/omise/android/ParcelableTest.kt

```diff
@@ -0,0 +1,96 @@
+package co.omise.android
+
+import android.os.Bundle
+import android.os.Parcel
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import co.omise.android.models.*
+import org.joda.time.DateTime
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ParcelableTest {
+
+    @Test
+    fun testModelParceling() {
```

Comment:
Honestly this name bring me back to JUnit 3. Could you rename them?

Reply:
Not sure what you mean by JUnit 3 naming, what would we change it to?

Reply:
Could you check from other test classes?

Reply:
Remember we discussed about this before. 😂

Reply:
We did, you mean like `testModelParceling_Success`? I'll change them but I don't see the point adding that at the end just for the sake of adding it, since there are no different states for each test, they are all success states. Will make the changes

Reply:
Since JUnit 4 release it use `@Test` instead of put `test` prefix (it uses in JUnit 3) in test cases. So, it no point to put `test` as prefix.

Reply:
I see, ok I'll make those changes too 👌

Reply:
Done